luxonis_ml/data/datasets/luxonis_dataset.py

```diff
@@ -154,15 +154,20 @@ def _init_paths(self) -> None:
                 self.bucket_storage, self.bucket, self.team_id, self.dataset_name
             )
 
-    def _load_df_offline(self) -> Optional[pl.DataFrame]:
+    def _load_df_offline(
+        self, lazy: bool = False
+    ) -> Optional[pl.LazyFrame | pl.DataFrame]:
```

Comment:
Should use `Union` for python3.8 tests to pass: `Optional[Union[pl.LazyFrame,pl.DataFrame]]`

Reply:
Made changes in the '_Improved type annotations_' commit.